NestJS/src/Templates/template.listener.ts

```diff
@@ -13,8 +13,19 @@ export class TemplateListener {
       queue.activateConsumer((message) => {
         var msg = message.getContent()
         var data = JSON.parse(msg)
-        console.log(msg)
-        templateService.create(data)
+        switch(data.task){
+            case "create":
+            templateService.create(data)
+            console.log("Successfully created new template")
+            break;
+            case "update":
+            templateService.update(data)
+            console.log("Successfully updated the template")
+            case "delete":
+            templateService.deleteOne(data.id)
+            console.log("Successfully deleted the template")
```

Comment:
there is no break on 'update' branch, also there is no default branch
also remember that there might be raised exceptions
consider to use strategy pattern

Reply:
Ok, got it! Thanks!